commands/register.js

```diff
@@ -194,6 +191,10 @@ function chooseAspect(input,client,message){
     aspectChoice=input.toUpperCase();
     }
   }
+  //sets the aspect based on registration choice (TO BE REMOVED?)
```

Comment:
This comment was only phrased like this because one of our later on plans is to require you to configure your classpect and other data pre-game in userdata (which is why that's created on your first message to the bot, pre-registration). If you look at it, it has fields for aspect and class. So it'll stay this way until we implement more pre-game set up.